README.md

```diff
@@ -195,6 +195,52 @@ analysis = FileAnalysis(file_hash=<file_sha256>)
 analysis.send(wait=True, wait_timeout=datetime.timedelta(minutes=1))
 ```
 
+#### Analyses History
+ - File
+
+```python
+history_results = query_file_analyses_history(
+    start_date = datetime.datetime.now() - datetime.timedelta(days=3),
+    end_date: datetime.datetime.now(),
+    api = api,
+    aggregated_view: False,
+    sources=["xsoar"],
+    verdicts=['malicious'],
+    file_hash='8d1131e418bdca5fb1abbb270c7cab46f169babcf7417cbf8557d2f3fe8e6b86',
```

Comment:
let's change it to <file_hash> as we do in other places in this readme file (for all the values)